lib/lower/cfg-builder.h

```diff
@@ -0,0 +1,352 @@
+//===-- lib/burnside/cfg-builder.h ------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef FORTRAN_LOWER_BRIDGE_CFG_BUILDER_H_
+#define FORTRAN_LOWER_BRIDGE_CFG_BUILDER_H_
+
+/// Traverse the AST and complete the CFG by drawing the arcs, pruning unused
+/// potential targets, making implied jumps explicit, etc.
+class CfgBuilder {
+
+  AST::Evaluation *getEvalByLabel(const Pa::Label &label) {
```

Comment:
Where is Pa::Label coming from here? This file has no includes.

Reply:
The file should only be included in one place. It's implementation, not interface, despite the filename.